app/models/account_migration.rb

```diff
@@ -12,6 +12,7 @@ class AccountMigration < ApplicationRecord
   after_create :lock_old_user!
 
   attr_accessor :old_private_key
+  attr_writer :old_person_handle
```

Comment:
I would prefer `old_person_diaspora_id` here, because `handle` is the old naming.